mock-caas/src/main/resources/application.yml

```diff
@@ -13,4 +13,5 @@ auth:
     ccm.enable: false
     baseimage.enable: true
     event-generation:
-      expiration-minutes: 10
\ No newline at end of file
+      expiration-minutes: 10
+    freeipa.ha.enable: true
```

Comment:
this means that in local mode we will always use ha IPA right ? I dont think that this is required in local dev

Reply:
It means that when HA is requested it can be provisioned. It doesn't require it. The choice of HA vs non-HA is made in the create environment request.